jac-uk/apply
# 2091/self assessment document extraction

## What's included?
Changes to the way that self assessments work.
Candidates now download the template which is uploaded by the team, fill it out, and submit the form, at which point they are presented with the extracted content minus the template to confirm. 
  
Closes jac-uk/admin#2091
  
## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

[Example exercise](https://jac-apply-develop--pr1058-2091-self-assessment-xb2yt24w.web.app/vacancy/IUSNh8uyhmM3u4x7F8eC/)

- start an application for the exercise
- go to self assessment with competencies
- download and fill out the template (write anything - **do not remove template content** _only add your answers_ )
- upload your filled out template
- submit the form
- ensure the extracted content is accurate
- submit the content 
- ensure the done flag is next to self assessment 

- download and fill out the template
- Add extra content, remove template content
- upload your filled out template
- submit the form
- view the extracted content
- amend and submit the content 
- ensure the done flag is next to self assessment 

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
